samples/Common.c

```diff
@@ -146,12 +146,10 @@ STATUS masterMessageReceived(UINT64 customData, PReceivedSignalingMessage pRecei
                                                        pReceivedSignalingMessage->signalingMessage.peerClientId,
                                                        TRUE,
                                                        &pSampleStreamingSession));
+        pSampleStreamingSession->firstSdpMsgReceiveTime = GETTIME();
         pSampleConfiguration->sampleStreamingSessionList[pSampleConfiguration->streamingSessionCount++] = pSampleStreamingSession;
     }
 
-    MUTEX_UNLOCK(pSampleConfiguration->sampleConfigurationObjLock);
```

Comment:
deliberate change? Not sure why it was part of this PR.

Reply:
the lock is unlocked at the end of the function. The entire function needs to be locked because each new signaling message call is a new thread.